src/components/EventList.js

```diff
@@ -58,6 +60,9 @@ export const EventList = ({
           return <LoadingSpinner loading={!listEndReached} />;
         }
 
+        if (!data?.length) {
+          return <EmptyPage title="Im Moment konnte kein anzuzeigender Inhalt gefunden werden." />;
+        }
```

Comment:
interesting, works, right? without checking deeper i haven't think of puttin it in the footer. have you tried `ListEmptyComponent` of the react native section list? https://reactnative.dev/docs/sectionlist#listemptycomponent

maybe this fits more in our use case?